buddy-guide.md

```diff
@@ -7,24 +7,20 @@
 - [End of Week 1](#end-of-week-1)
 
 # Buddy Guide
-
-
+As part of Red Hat's internship programme, each new hire will be allocated a buddy. A buddy is an experienced team member who will be able to provide training, instruction and leadership for the student. This is a guide for what is expected of you as a buddy to ensure that the student have a pleasant experience during their internship.
 
 
 # Before the new student starts
-
-
+- Ensure you are familiar with the student you are paired up with. Read their CV, learn something about them. 
```

Comment:
@JameelB We decided that we would only include content in this repo that would be used by the student/intern. This is more internal stuff for ourselves

Reply:
Probably a need for 2 docs - a buddy guide which would outline the duties/ responsibilities of a buddy, and a student guide, which would include a section on what the student could expect from their buddy.

Reply:
@damienomurchu yes agreed. it would suit the scope of this repo to update this to `what to expect from your buddy`